website/docs/general/join.md

```diff
@@ -0,0 +1,36 @@
+---
+id: join
+title: Join the conversation
+keywords:
+  - Slack
+  - Community
+  - Apache APISIX
+description: Join the Apache APISIX community and get involved in the conversations.
+---
+
+
+Apache APISIX is a project maintained by community members under the Apache Software Foundation and abide by [ASF's code of conduct](https://www.apache.org/foundation/policies/conduct.html).
+
+We have a transparent and inclusive community where you can join in and contribute to the discussion.
+
+### Subscribe to the Mailing List
+
+You can subscribe to the Apache APISIX mailing list to discuss issues, suggest new ideas, and participate in other community discussions.
+
+1. To subscribe to the mailing list, first, send an email to dev-subscribe@apisix.apache.org.
+
+2. Once you send the email, you will receive a confirmation e-mail from dev-help@apisix.apache.org.
+
+3. Reply to this email to confirm your subscription.
+
+4. Once you confirm, you will receive a welcome email with instructions on using and communicating over the mailing list.
+
+5. To view archived emails from the mailing list, check out the [history](https://lists.apache.org/list.html?apisix.apache.org).
+
+### Join the Slack Channel
+
+You can join the Apache APISIX Slack channel in two ways:
+
+- Using [this invite](https://join.slack.com/t/the-asf/shared_invite/zt-vlfbf7ch-HkbNHiU_uDlcH_RvaHv9gQ) (_Please open an issue if this link is expired_).
```

Comment:
How about linking an `Issue Create Page` here?

Reply:
IG that would be nice